src/compare-page-pull-request/compare-page-pull-request.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { h } from 'dom-chef';
+import elementReady from 'element-ready';
+import { getRepoURL } from '../page-detect';
+import 'selector-observer';
+
+const Constants = {
+    COMPARE_DETAILS_SELECTOR: '#branch-compare.content-container'
+};
+
+export default async function init() {
```

Comment:
I like to name the exported default function the same way it is going to be named when imported if possible, so I would rename this to `comparePagePullRequest`, since this is how you called it in `main.js`

Reply:
thats fair